Bottom: cadd74466e1decfc85f53b1c377daff60f7961f4
Top:    bf810d8f9d72f88df96f2880b52edb7d1aae5b98
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-18 20:53:20 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
